app/RouteHandlers/AddDonationHandler.php

```diff
@@ -221,4 +223,14 @@ private function newTrackingInfoFromRequest( Request $request ): DonationTrackin
 	private function filterAutofillCommas( string $value ): string {
 		return trim( preg_replace( ['/,/', '/\s{2,}/'], [' ', ' '], $value ) );
 	}
+
+	/**
+	 * Since the application has no concept of sessions or logging in / out, we reset session data here
```

Comment:
The comment has now become contradictory. Can you please state why this exists, e.g. "Unset 'address was changed' state in session, to display the right state on teh confirmation page."